### DISCUSSION TITLE
Enum collector

### DISCUSSION THREAD

**Discussion by innocenzi** [spatie/typescript-transformer#41] [Ideas]
Hey,

I'm wondering if there is a reason not to have an `EnumCollector` provided by default, since there is an `EnumTransformer`. If there is no specific reason, can I PR one so I can stop using `#[TypeScript]` on my enums?

Thanks! ❤️

**Comment by rubenvanassche** [comment]
Yeah sure!

**Reply by innocenzi** [reply]
Cool thanks! I'll work on it when I have some time.
